docs/components/operate/index.md

```diff
@@ -19,13 +20,16 @@ dark: useBaseUrl('img/operate-introduction_dark.png'),
 
 In addition to providing visibility into active and completed process instances, Operate also makes it possible to carry out key operations such as resolving <!-- FIXME: [incidents](/reference/incidents.html) --> incidents, and updating process instance variables.
 
+[//]:# (What is the link to fix here?)
```

Comment:
@santanaluiz any idea what link should go in here?

Reply:
I would guess here:
https://docs.camunda.io/docs/components/operate/userguide/resolve-incidents-update-variables/

no?